render/egl.c

```diff
@@ -418,17 +416,6 @@ bool wlr_egl_destroy_image(struct wlr_egl *egl, EGLImage image) {
 	return egl->procs.eglDestroyImageKHR(egl->display, image);
 }
 
-EGLSurface wlr_egl_create_surface(struct wlr_egl *egl, void *window) {
-	assert(egl->procs.eglCreatePlatformWindowSurfaceEXT);
-	EGLSurface surf = egl->procs.eglCreatePlatformWindowSurfaceEXT(
```

Comment:
I've slightly edited this commit to remove the `eglCreatePlatformWindowSurfaceEXT` field from `struct wlr_egl`.

Reply:
Thanks for doing that and thanks for merging my PR! I was thinking of removing that in a separate PR, but now I don't have to.